spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
@chrisarcand 
> Can you give a more specific example?
#13632 is one of the examples

(btw, don't know why your the other